### ISSUE TITLE
blockquotes: no nested markdown?

### ISSUE THREAD

**Issue by pixelass** [patternplate/patternplate#174]
It seems like blockquotes in markdown files don't allow nested markdown.

> Blockquotes can contain other Markdown elements, including headers, lists, and code blocks: - [source](https://daringfireball.net/projects/markdown/syntax#blockquote)

## Code

```markdown
> **Relative values**  
> They can be used for slides and fragments  
> `translateX(var(--time) * 100%)`  
>
> **Viewport values**  
> They can be used for slides and in rare cases for fragments  
> `translateX(var(--time) * 100vw)`  
>
> **Absolute values**  
> They can be used for fragments and in rare cases for slides  
> `translateX(var(--time) * 100px)`
```

## Expected

> **Relative values**  
> They can be used for slides and fragments  
> `translateX(var(--time) * 100%)`  
>
> **Viewport values**  
> They can be used for slides and in rare cases for fragments  
> `translateX(var(--time) * 100vw)`  
>
> **Absolute values**  
> They can be used for fragments and in rare cases for slides  
> `translateX(var(--time) * 100px)`

### HTML

```html
<blockquote class="markdown-blockquote__exports-bIAKvV eeLCvo text__StyledText-elEkvz dfvBTr">
Relative values
They can be used for slides and fragments
translateX(var(--time) * 100%)  
Viewport values
They can be used for slides and in rare cases for fragments
translateX(var(--time) * 100vw)  
Absolute values
They can be used for fragments and in rare cases for slides
translateX(var(--time) * 100px)
</blockquote>
```
### Screenshot
<img width="761" alt="screen shot 2018-03-13 at 20 21 24" src="https://user-images.githubusercontent.com/1148334/37364934-30deaade-26fc-11e8-9b99-4a57b40cda4a.png">

## Version

```
node_modules/.bin/patternplate --version
2.0.0-55
```

**Comment by marionebl** [comment]
I think we do not handle nested markdown all that well, currently. Fix for this will have to happen in https://github.com/patternplate/patternplate/blob/master/packages/components/src/markdown/index.js#L32-L79

**Comment by marionebl** [comment]
Fixed via `2.0.0-57`
